src/main/java/com/project/serverpool/service/ServerService.java

```diff
@@ -0,0 +1,138 @@
+package com.project.serverpool.service;
+
+import com.project.serverpool.database.AerospikeDatabase;
+import com.project.serverpool.domain.Client;
+import com.project.serverpool.domain.Server;
+import com.project.serverpool.model.dao.daoImpl.ClientDao;
+import com.project.serverpool.model.dao.daoImpl.ServerDao;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+@Service
+public class ServerService {
+
+    private final ServerDao serverDao = new ServerDao();
+    private final ClientDao clientDao = new ClientDao();
```

Comment:
since I did not use (Jpa) for aerospike (i did not know there is a dependence for it, I searched and found one "spring-data-aerospike-starters") and manually create it, I can not apply constructed injection here because ServerDan and ClientDao are not beans, so spring starts the program, it cant inject both Dao objects.

I used the final reference here for thread safety because once assigned reference of an object it cannot point to another object.